_dev/eslint-plugin-fxa/test/no-exclusive-tests.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
same question as above. Not sure if this needs our MPL copyright header, or if it was pulled from somewhere else.